test/test_iron_cache.rb

```diff
@@ -171,7 +171,7 @@ def test_incrementors
 
   def test_size
     cache = @client.cache("test_size")
-    cache.clear
+    clear_cache(cache.name)
```

Comment:
If cache does not exist, it raises error and test fails. But is not expected behaviour.